docs/ftue/02_translator/FTUE_02_translator.md

```diff
@@ -89,37 +114,48 @@ Set up the translator agent:
 
 To see the final translation:
 
-1. Connect the output of the second agent to the display text node
-1. When the workflow runs, this node will show the translated English text
+1. Connect the output of the second Agent to the DisplayText node
+1. Run your workflow.
+1. After the workflow runs, this node will show the translated English text:
 
 <p align="center">
-  <img src="../assets/mandarin_display.png" alt="Display setup"  width="500">>
+  <img src="../assets/mandarin_display.png" alt="Display setup"  width="500">
 </p>
 
-## Understand Execution Order (Exec Chain)
+## Understanding Execution Order (Exec Chain)
 
-A key concept in Griptape Nodes is the execution chain:
+A key concept in Griptape Nodes is the execution chain. As workflows become more complex, controlling the order of execution becomes important. Let's explore this concept.
 
 1. Notice the "exec in" and "exec out" pins (half-circle connectors) on nodes
 1. These define the order in which nodes run
-1. For complex workflows, connect the last pin of one section to the first pin of the next
-1. This ensures nodes run in the correct sequence, even with complex data flows
+1. For complex workflows, connect the exec ports in the order you want execution to occur
+1. This ensures nodes run in the intended sequence, even with complex data flows
 
+```
 <p align="center">
```

Comment:
Oops! this is showing up as html - I _think_ it's because it needs another space after the triple backtick on the line before..

![image](https://github.com/user-attachments/assets/eece43db-1d62-428e-80eb-505fd568ff53)